playground/package.json

```diff
@@ -4,8 +4,7 @@
   "scripts": {
     "dev": "nuxi dev",
     "build": "nuxi build",
-    "start": "nuxi preview",
-    "generate": "nuxi generate"
```

Comment:
@danielroe I'm curious, how come we don't want this in the package.json?

Reply:
Perfectly fine to have it, but I try to restrict changes in PRs to the single thing the PR is doing.